plugins/foreman_tasks/0.8/index.md

```diff
@@ -0,0 +1,302 @@
+- **State** - one of *running*, *stopped*, *paused*, running tasks are tasks that are actively being performed. If an error occurres during the run, based on the task rescue strategy, it can be paused or switched to stopped. Paused tasks can be resumed making them running again. After all task steps are finished the task state is stopped. Stopped tasks can't be resumed. 
+- **Triggered by** - user who started the task
+- **Execution type** - possible values are *immediate* and *delayed*, delayed tasks are usually scheduled to start at specific time in future, to determine the execution type, attributes *start_at* and *started_at* are compared
+
+- **Start at** - indicates the time when the task should start
+- **Started at** - indicates the time when the task really started
+- **End at** - indicates the time when the task stopped
+- **Start before** - indicates the deadline after which the task should not start if it hasn't started yet
+
+- **Progress** - the task may consist of multiple steps or sub-tasks, the progress is calculated based on percentage of finished steps/sub-tasks
+
+Note that you get more information by hovering some of the attribute values.
+
+### 3.2.1 Results and State related Q&A
+
+- Q: Why I see paused task?
+- A: Most common cause is that the task was programmed to pause on errors and some error occurred. On the other hand, there might be tasks that are paused because they are waiting to be resumed by some external service when it's ready. To find out more information, check if you see *Errors* field on task page or red box under the Errors tab. If you see something, the pause was very likely caused by an error.
+
+- Q: What is the pending result exactly?
+- A: While the task is in progress we can't tell what is the result, in such case the result is *pending*
+
+- Q: Where do I find the error that I should report if the task failed?
+- A: The best place to look is the task detail page. In the first tab, you will see the human readable message based on the exception that was raised. The format depends on author of the task. If the task is based on Dynflow, you can find more information by navigating to the Errors tab on the same page. If you report an issue, make sure you attach the content of this tab with other logs. Note that the input and output fields may contain sensitive data so make sure you clean all such data while uploading it.
+
+- Q: There is a task with state *paused* and result *error*, what does that mean?
+- A: This indicates that the task was paused because of some error occurred. In task detail you should see question mark at *Result* field but in table, we display the value matching to what the result of the task would be, if we stoped it right now. Thanks to this you can also easily search tasks that needs attention by filtering on result value. The same applies for warning result. The pending value is displayed only if we don't have any information yet.  
```

Comment:
I think the task might be paused even if no error occurred so you would see paused task with success state. @iNecas or @adamruzicka do you have any example? I think "Listen on candlepin events" was such task but I think the state was renamed there to be less confusing? The generic use case is that the task triggers some external event and pauses itself. Pause actually means it's waiting to be resumed by some external system, e.g. after the external event finished.

Anyway this Q is about specific combination of paused task with error result. As it says, in task detail, we denote this task result as pending since the task hasn't finished yet, but in task list we display the "current status". I'd prefer to always display pending until the task is stopped but it was not accepted. The reason was that this information might be useful even before the task finished.